source/extensions/filters/http/oauth2/oauth_client.cc

```diff
@@ -66,6 +66,7 @@ void OAuth2ClientImpl::onSuccess(const Http::AsyncClient::Request&,
   const auto response_code = message->headers().Status()->value().getStringView();
   if (response_code != "200") {
     ENVOY_LOG(debug, "Oauth response code: {}", response_code);
+    ENVOY_LOG(debug, "Oauth response body: {}", message->bodyAsString());
```

Comment:
I'm afraid of the amount of data that would be logged here.
Wouldn't this even make debug unusable in some cases?
Eg: big pages being proxied

Reply:
Let me clarify this @joaovitor. What is being logged here is the response body of the request to the OAuth provider. This is not the upstream request (to service being proxied in the end).

Reply:
Great @andreyprezotto ... Hope to see this PR merged.
This will improve oauth integration in envoy a lot.
Thanks for this work!